Documentation/genindex.rst

```diff
@@ -0,0 +1,7 @@
+.. include:: /Includes.rst.txt
```

Comment:
This currently produces an empty page...

Reply:
Yes, this is by design and will display a nice placeholder text with the next documentation rendering toolchain release, see https://github.com/t3docs/docker-render-documentation/issues/124.

Reply:
Ah great. Should I maybe add some keywords to the docs to get here something useful?

Reply:
Btw I have to extens the install page a bit, there is a pending issue with v11. So I also can do some additional changes...

Reply:
Sure, you could merge this PR and then add additional indexes and texts. This PR could be about the transformation, and yours about enhancements.

Reply:
Yes, that's my plan.